template1.css

```diff
@@ -0,0 +1,144 @@
+* {
+  margin: 0px;
+  padding: 0px;
+  font-family: "Times New Roman", Times, serif;
+  -webkit-print-color-adjust: exact !important;
+}
+.container {
+  margin-left: auto;
+  margin-right: auto;
+}
+.main {
+  display: grid;
+  grid-template-columns: 44% 1% 55%;
+  padding-left: 2%;
```

Comment:
Please always try px insisted of %.

Reply:
tried pixels but % is working much better